### PR TITLE
material ui text fields and icon uploaded

### PR DESCRIPTION
**Pull request by tommysullivan**
Screen Shot 2019-06-26 at 3 09 58 PM
Screen Shot 2019-06-26 at 3 10 03 PM
Screen Shot 2019-06-26 at 3 10 10 PM

### CODE REVIEW

**Comment by bob-bins** [comment]
hmm `http://localhost:3002/organizations/new` goes to a blank page for me. Can you double check that it still works for you?

**Comment by tommysullivan** [comment]
> hmm `http://localhost:3002/organizations/new` goes to a blank page for me. Can you double check that it still works for you?

@bob-bins yeah it works for me... did u do a fresh `hp clientDev` and refresh? not sure.

**Comment by bob-bins** [comment]
Yeah i did. I'll try again today. But the code looks good to me!

**Comment by tommysullivan** [comment]
@bob-bins @PatrickMetsch i spent lots of time prettying things up today, killing bootstrap, using material ui constructs more, using the grid, added tooltips, killed the image uploader, etc. 

please check it out. ready to merge.

File: src/components/forms/FilePicker.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export const FilePicker = ({onChange, acceptedContentTypes = "*", children}) => <Fragment>
+    <input
+        accept={acceptedContentTypes}
+        style={{display: "none"}}
+        id="contained-button-file"
+        multiple={false}
+        type="file"
+        onChange={fileUploadHandler(onChange)}
+    />
+    <label htmlFor="contained-button-file">
```
Comment (tommysullivan) [inline src/components/forms/FilePicker.tsx]: can you make id a guid?

File: src/components/homePage/HomePage.tsx [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
-                <h2 style={{textAlign: "center", marginTop: "10px"}}>
-                    <a href="mailto:info@heartpoints.org">info@heartpoints.org</a> 
-                </h2>
-            </div>
-        </div>
-    </div>
-</div>
-}
\ No newline at end of file
+export const HomePage = () => <Container className="vertical-center">
```
Comment (tommysullivan) [inline src/components/homePage/HomePage.tsx]: can we move this css class out of the html file

**Pull request merged**